airbyte-config/init/src/main/resources/seed/source_specs.yaml

```diff
@@ -2156,6 +2156,30 @@
     supportsNormalization: false
     supportsDBT: false
     supported_destination_sync_modes: []
+- dockerImage: "airbyte/source-convex:dev"
```

Comment:
TODO: i think i need to delete this before sending to airbyte for review, but it's necessary for running locally